### PR TITLE
Revise TransposeImage

### PR DESCRIPTION
**Pull request by Toraudonn**
- Fixed a bug in the transpose operation inside `TransposeImage` (#163)
- For future reference, I've added `TransposeObs` as a base class to build upon other transpose operations other than images
- Added `_set_op` function in `TransposeImage`

### CODE REVIEW

**Comment by ikostrikov** [comment]
Looks good to me! @timmeinhardt what do you think?

**Comment by timmeinhardt** [comment]
The coding looks good. However, it is limited to certain transpositions and observations that have a notion of an image, i.e., height, width and channel. 

The class could receive just a tuple with indices which then looks something like this:
`obs = TransposeObs(obs, (2,0,1))`

**Comment by ikostrikov** [comment]
@timmeinhardt I agree but I think it's ok to merge the pull request in the current form since a more general wrapper would require more work.

**Comment by ikostrikov** [comment]
@Toraudonn I've noticed a minor problem. chw or hwc depends on inputs. I would prefer to keep the name TransposeImage (since dim=3) and send a list of dims instead of ops.

File: a2c_ppo_acktr/envs.py [hunk 1]
```diff
@@ -118,18 +118,41 @@ def observation(self, observation):
...
+class TransposeObs(gym.ObservationWrapper):
     def __init__(self, env=None):
+        """
+        Transpose observation space (base class)
+        """
+        super(TransposeObs, self).__init__(env)
+
+
+class TransposeImage(TransposeObs):
+    def __init__(self, env=None, op="chw"):
```
Comment (ikostrikov) [inline a2c_ppo_acktr/envs.py]: Replace op with a list of indices.

Comment (Toraudonn) [reply a2c_ppo_acktr/envs.py]: Done!

**Comment by ikostrikov** [comment]
Thanks!

Looks good to me. Merged.

**Pull request merged**